### ISSUE TITLE
Hard to find download button

### ISSUE THREAD

**Issue by dzmitry-lahoda** [gitextensions/gitextensions#6252]
1. Visit https://github.com/gitextensions/gitextensions/releases/tag/v3.00.00
2. Try to click all links on the top of the page. 
3. Non of them are download.
4. Find main site. Find loanword button. 
5. Get on https://github.com/gitextensions/gitextensions/releases/tag/v3.00.00 again.
6. May be scroll down?
7. Got it.

I would expect download button on top of page or other means easily discoverable.

**Comment by vbjay** [comment]
See the assets? Those are the downloads. We are not in control of how
GitHub formats that page.

**Comment by vbjay** [comment]
![screenshot_20190215-071253](https://user-images.githubusercontent.com/3629489/52856082-6e762980-30f1-11e9-8f51-4041d4edd14a.png)

**Comment by pmiossec** [comment]
I think we could add a "Download the last version" link towards https://sourceforge.net/projects/gitextensions/files/latest/download in the README.md.
Even if the link doesn't point to the last version 😢 I don't know how sourceforge works and if something should be done to fix it. 

And that's also right that the "version 3.0x - NOW available!" in https://github.com/gitextensions/gitextensions#current-status points to an outdated version 😢 We should use https://github.com/gitextensions/gitextensions/releases/latest as link.

But that don't work also because https://github.com/gitextensions/gitextensions/releases/tag/v3.0.1 is a pre-release. Could you fix that @RussKie ?

**Comment by RussKie** [comment]
I find the SourceForge website abhorrent... It injects the code into our release page via a setup integration but I'm not quite sure how artifacts are managed on the SF side.

The point is - we could make the download link more prominent both on the front page (via the README) and on the release page.

**Comment by dzmitry-lahoda** [comment]
3.0.0 release has very long release notes. I have tried to click all buttons on top of the page to download. 3.0.1 also has more than a page notes on my 1920x1080 15'' display, but I guess would scroll that out. For some reason, I have always easily downloaded other applications' releases on github.  May be all had short release notes.

I have checked PowerShell releases - they have put it onto main page. But if to open github release - long notes either. So put https://github.com/gitextensions/gitextensions/releases/download/v3.00.00/GitExtensions-3.00.00.4433.msi onto main page readme or other md file linked from readme is possible approach. Most people just download and dot not care of notes. So notes(current link which is used as release) and donation link could be on that page.

![image](https://user-images.githubusercontent.com/757125/52895701-f0885000-31ce-11e9-8ad9-67f36fba6fc9.png)


![image](https://user-images.githubusercontent.com/757125/52895689-cdf63700-31ce-11e9-931f-3bcba0551062.png)
